.github/workflows/ci.yml

```diff
@@ -20,8 +20,10 @@ jobs:
       - uses: actions/checkout@v2
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@v1
-      - run: timeout 15m cargo test --workspace --all-features --all-targets
-
+      - run: |
+          timeout 5m cargo test -- --skip rpc --skip sequencer
```

Comment:
I'm not sure if I got the args here correctly. The aim was to run all tests excluding seq & rpc.

Reply:
I think the one you ended up with is better than this original you flagged.